cluster/images/hyperkube/kube-proxy.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
IIRC, it's necessary to pass `--resource-container=""` for kube-proxy in this case. 
Ref [#17619(comment)](https://github.com/kubernetes/kubernetes/issues/17619#issuecomment-160779388)